sph/include/sph/iad.hpp

```diff
@@ -45,26 +45,26 @@ void computeIADImpl(size_t startIndex, size_t endIndex, size_t ngmax, Dataset& d
     for (size_t i = startIndex; i < endIndex; ++i)
     {
         size_t ni = i - startIndex;
-        kernels::IADJLoop(i,
-                          sincIndex,
-                          K,
-                          box,
-                          neighbors + ngmax * ni,
-                          neighborsCount[i],
-                          x,
-                          y,
-                          z,
-                          h,
-                          m,
-                          ro,
-                          wh,
-                          whd,
-                          c11,
-                          c12,
-                          c13,
-                          c22,
-                          c23,
-                          c33);
+        kernels::IADJLoop3L(i,
```

Comment:
I've started to use the "plain" names for the VE functions and instead add the suffix "3L" to the old kernels
where needed to avoid name clashes.